files/chef-marketplace-gem/lib/marketplace/reckoner/metrics/erchef_crashes.rb

```diff
@@ -0,0 +1,17 @@
+require 'marketplace/reckoner/metrics/base'
+
+class Marketplace::Reckoner::Metrics
+  class ErchefCrashes < LogParser
+    def filenames
+      [
+        '/var/log/opscode/opscode-erchef/crash.log',
```

Comment:
This might collide with log rotation - if these fill up quickly they could rotate out within a day.

Also for all the logs the number of files are configurable.  I think we need to decide if we want to log metrics, and if so pull in a better log parser.

But this shouldn't block us.